mods/ts/maps/fields-of-green/map.yaml

```diff
@@ -0,0 +1,1459 @@
+		Location: 87,-12
+	Actor407: gasand
+		Owner: GDI
+		Location: 88,-12
+	Actor414: gasand
+		Owner: GDI
+		Location: 81,-12
+	Actor409: gasand
+		Owner: GDI
+		Location: 89,-12
+	Actor410: gasand
+		Owner: GDI
+		Location: 89,-10
+	Actor411: gasand
+		Owner: GDI
+		Location: 89,-11
+	Actor412: gasand
+		Owner: GDI
+		Location: 89,-9
+	Actor413: gasand
+		Owner: GDI
+		Location: 89,-8
+	Actor415: gasand
+		Owner: GDI
+		Location: 80,-12
+	Actor416: gasand
+		Owner: GDI
+		Location: 79,-12
+	Actor417: gasand
+		Owner: GDI
+		Location: 87,0
+	Actor418: gasand
+		Owner: GDI
+		Location: 88,0
+	Actor419: gasand
+		Owner: GDI
+		Location: 89,0
+	Actor420: gasand
+		Owner: GDI
+		Location: 89,-1
+	Actor421: gasand
+		Owner: GDI
+		Location: 89,-2
+	Actor422: gasand
+		Owner: GDI
+		Location: 80,-2
+	Actor423: gasand
+		Owner: GDI
+		Location: 79,-2
+	Actor424: gasand
+		Owner: GDI
+		Location: 78,-2
+	Actor425: gasand
+		Owner: GDI
+		Location: 76,-3
+	Actor426: gasand
+		Owner: GDI
+		Location: 76,-2
+	NodW: waypoint
+		Owner: Neutral
+		Location: 40,5
+	GDIW: waypoint
+		Owner: Neutral
+		Location: 86,-8
+	MidW: waypoint
+		Owner: Neutral
+		Location: 62,5
+	TopW: waypoint
+		Owner: Neutral
+		Location: 56,-18
+	BotW: waypoint
+		Owner: Neutral
+		Location: 59,27
+	Actor429: e1
+		Owner: GDI
+		Location: 85,-5
+		SubCell: 2
+		Facing: 92
+		TurretFacing: 92
+	Actor440: e1
+		Owner: GDI
+		Location: 86,-10
+		SubCell: 2
+		Facing: 92
+		TurretFacing: 92
+	Actor431: e1
+		Owner: GDI
+		Location: 85,-7
+		SubCell: 2
+		Facing: 92
+		TurretFacing: 92
+	Actor432: e2
+		Owner: GDI
+		Location: 83,-8
+		SubCell: 2
+		Facing: 92
+		TurretFacing: 92
+	Actor433: e2
+		Owner: GDI
+		Location: 83,-9
+		SubCell: 2
+		Facing: 92
+		TurretFacing: 92
+	Actor434: e1
+		Owner: Nod
+		Location: 39,6
+		SubCell: 2
+		Facing: 92
+		TurretFacing: 92
+	Actor439: e1
+		Owner: Nod
+		Location: 39,11
+		SubCell: 2
+		Facing: 92
+		TurretFacing: 92
+	Actor436: e1
+		Owner: Nod
+		Location: 40,6
+		SubCell: 2
+		Facing: 92
+		TurretFacing: 92
+	Actor437: e3
+		Owner: Nod
+		Location: 37,3
+		SubCell: 2
+		Facing: 92
+		TurretFacing: 92
+	Actor438: e3
+		Owner: Nod
+		Location: 37,4
+		SubCell: 2
+		Facing: 92
+		TurretFacing: 92
+	North: waypoint
+		Owner: Neutral
+		Location: 41,-33
+	South: waypoint
+		Owner: Neutral
+		Location: 67,48
+	GDIBase: waypoint
+		Owner: Neutral
+		Location: 80,-1
+	NodBase: waypoint
+		Owner: Neutral
+		Location: 45,6
+
+Rules:
+	Player:
+		-ConquestVictoryConditions:
+	World:
+		-CrateSpawner:
+		-StartGameNotification:
+		-SpawnMPUnits:
+		-MPStartLocations:
+		LuaScript:
+			Scripts: map.lua
+		ResourceType@Tiberium:
+			ResourceType: 1
+			ValuePerUnit: 0
+		MusicPlaylist:
+			BackgroundMusic: intro
+		GlobalLightingPaletteEffect:
+			Red: 1.0
+			Green: 1.0
+			Blue: 0.7
+			Ambient: 0.7
+	HARV:
+		-Targetable:
+	GALITE:
+		-Targetable:
+	APC:
+		Cargo:
+			InitialUnits: e1, e1, e2, e2, medic
+	GACTWR:
+		Health:
+			HP: 99999999
```

Comment:
Couldn't we just make those buildings invulnerable using damage modifiers (like the ra shellmap does)?